packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+
+### Changed
+
+- **BREAKING** split `NotificationServiceController` constructor and initialisation methods ([#5504](https://github.com/MetaMask/core/pull/5504))
+  - This makes the controller easier to use with the Modular Controller Initialisation architecture,
+    but requires the method call `.init()` to finialise initialising the controller.
```

Comment:
Happy to reword this.

Context:

The `NotificationServicesController` called external actions inside of its constructor. This makes it very hard to adopt the new Modular Controller pattern, as it relies on certain controllers to be created before it can be created.

This new change means that we can use the new pattern to construct our controllers, and then call `.init()` once all controllers have been initialised.